Formula/qt5.rb

```diff
@@ -90,12 +100,32 @@ def install
       -qt-pcre
       -nomake tests
       -no-rpath
+      -pch
+      -system-proxies
+      -optimized-tools
```

Comment:
`-pch` isn't on by default? If `-optimized-tools` is on by default we can omit it. `-system-proxies` seems sensible.